### PR TITLE
Refactor/replace web3 with ethers

### PR DESCRIPTION
**Pull request by KimiWu123**

### CODE REVIEW

File: packages/cli/src/apps/coordinator/cli.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ const main = async () => {
   Object.assign(config, argv)
   // Load keystore if needed
   if (config.keystoreFile) {
-    if (config.keystore) {
+    if (config.encryptedKeystore) {
```
Comment (sifnoc) [inline packages/cli/src/apps/coordinator/cli.ts]: Would you update for this on config files in `packages/cli/*.json`
for example, from `keystore` to `encryptedKeystore`.
https://github.com/wanseob/zkopru/blob/a5a7d0cb83e3d2f6afc987fa3ce25db2f981be4b/packages/cli/coordinator.json#L16-L20
still `config.keystore` in it.

File: packages/core/src/validator/offchain/offchain-utxo-tree-validator.ts [hunk 2]
```diff
@@ -137,7 +139,7 @@ export class OffchainUtxoTreeValidator extends OffchainValidatorContext
     )
     if (!computedRoot.eq(block.header.utxoRoot.toBigNumber())) {
       // slashable
-      const proofId = randomHex(32)
+      const proofId = crypto.randomBytes(32).toString('hex')
```
Comment (sifnoc) [inline packages/core/src/validator/offchain/offchain-utxo-tree-validator.ts]: This cause error on handling slashable block in fullnode-validator

```
Error: invalid BigNumber string (argument="value", value="a2e0a97fb4c1cf59be33056731e20c1607193a668e01be4bade98cc28b7ed03f", code=INVALID_ARGUMENT, version=bignumber/5.6.2)
```

Was there an error by this on test cases?

File: packages/cli/src/apps/coordinator/cli.ts (hunk 1 above)
Comment (KimiWu123) [reply packages/cli/src/apps/coordinator/cli.ts]: rollback to `keystore`, fixed in [5ddcdb7](https://github.com/wanseob/zkopru/pull/10/commits/5ddcdb7f0921dbe40ec9d08eae7532b035ad0fac)

File: packages/core/src/validator/offchain/offchain-utxo-tree-validator.ts (hunk 2 above)
Comment (KimiWu123) [reply packages/core/src/validator/offchain/offchain-utxo-tree-validator.ts]: could I know which test case? It all passed on my side.

Comment (sifnoc) [reply packages/core/src/validator/offchain/offchain-utxo-tree-validator.ts]: Actually, the error comes from a coordinator which was syncing to zkopru testnet on goerli.
the coordinator could not handle some error like above.

Comment (KimiWu123) [reply packages/core/src/validator/offchain/offchain-utxo-tree-validator.ts]: fixed in [b06eb15](https://github.com/wanseob/zkopru/pull/10/commits/b06eb15eed4d7833dd3b6ac2c4b33715588f2075)

**Review by sifnoc — APPROVED** [review]

**Comment by sifnoc** [comment]
This PR no need to merge if already merge #11 which is base this branch

**Pull request merged**